src/VirtoCommerce.Platform.Modules/AssemblyLoading/CustomAssemblyLoadContext.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace VirtoCommerce.Platform.Modules.AssemblyLoading
+{
+    public class CustomAssemblyLoadContext : AssemblyLoadContext
```

Comment:
Why do we need this type? It doesn't implement anything.

Reply:
because there is protected method LoadUnmanagedDllFromPath in the base class AssemblyLoadContext, look at (link)[https://docs.microsoft.com/en-us/dotnet/standard/assembly/unloadability]